frontend/src/components/NewRunParameters.test.tsx

```diff
@@ -37,6 +37,20 @@ describe('NewRunParameters', () => {
     expect(shallow(<NewRunParameters {...props} />)).toMatchSnapshot();
   });
 
+  it('clicking the open editor button for json parameters display an editor', () => {
```

Comment:
May I ask for one more test case covering also typing in the json Editor and submit the run?